src/components/Restoration/CorrectNameRequest.vue

```diff
@@ -1,59 +1,57 @@
 <template>
-  <v-form id="correct-nr-form" ref="correctNrForm" v-model="formValid" lazy-validation>
-    <!-- Dialogs -->
+  <v-form id="correct-name-request-form" ref="form" v-model="formValid" lazy-validation>
     <ConfirmDialogShared
       ref="confirm"
       attach="#app"
     />
 
     <v-row no-gutters>
       <v-col cols="1" class="mt-1">
-        <v-btn x-small fab outlined :ripple="false" color="gray7" class="step-icon" tabindex="-1">1</v-btn>
+        <v-chip class="step-icon">1</v-chip>
```

Comment:
Old:
![image](https://user-images.githubusercontent.com/10002889/218227567-b8cce14f-4fa9-49bc-99b6-7d1797e4bcbe.png)

New (and I'll update the style to match):
![image](https://user-images.githubusercontent.com/10002889/218227586-3a03c85c-2c76-4560-b6dc-fb476cae2af3.png)